src/testdir/test_tabline.vim

```diff
@@ -178,6 +179,9 @@ func Test_tabline_showcmd()
 
   call feedkeys("\<Esc>1234", "xt")
   call VerifyScreenDump(buf, 'Test_tabline_showcmd_2', {})
+
+  call feedkeys("\<Esc>:set tabline=\<CR>:\<CR>1234", "xt")
+  call VerifyScreenDump(buf, 'Test_tabline_showcmd_3', {})
 endfunc
```

Comment:
It seems that a `StopVimInTerminal(buf)` is missing in the end of this function, and it is also missing in the end of the last test function in `test_statusline.vim`

Reply:
I wasn't aware that helper function is required. I have added quite a few screendump tests at this point, it might be missing in all of them then. I don't think the test I copied it from the first time had this helper function.